markers/polarion.py

```diff
@@ -13,3 +14,11 @@ def extract_polarion_ids(item):
         return None
 
     return map(str, polarion.args)
+
+
+@pytest.fixture(autouse=True)
+def _update_polarion_in_junit(request, record_xml_property):
```

Comment:
I was wondering if this could be turned into a plugin instead of fixture.

Reply:
Can you explain the difference between the two?

Reply:
@mfalesni `record_xml_property` currently is not available as a api outside if fixtures

@shalomnaim1 as an hook we could transfer the metadata outside of test setup/teardown, which is desirable for more consistent data transfer

however a api for facilitating that isn't available currently

as soon as pytest has a fit api the data transfer can moved from a fixture to a runtestprotocol hook

Reply:
@RonnyPfannschmidt I see, thanks :-)